testrail.py

```diff
@@ -2,17 +2,21 @@
 import sys
 
 import pandas as pd
+import numpy as np
 
 from lib.testrail_conn import APIClient
+
 from database import (
     Database,
     Projects,
     TestSuites,
     ReportTestCaseCoverage,
+    ReportMilestones,
```

Comment:
I think this is fine for now.  At some point (if we start using data from many other sources: Jira, Bugzilla, github, TestRail, etc., we may wish to prefix those tables with source API, although that makes for longer table names.
Thus something like: `report_testcase_coverage` would become `report_testrail_testcase_coverage`, etc.

We can take a look at that when we review the CloudSQL DB